pages/dao/[symbol]/proposal/components/instructions/UXD/EditIdentityDepository.tsx

```diff
@@ -19,7 +19,7 @@ const schema = yup.object().shape({
     .min(0, 'Redeemable amount under management cap should be min 0'),
 });
 
-const RegisterMercurialVaultDepository = ({
```

Comment:
Wait... Oh we forgot to change the name. I think the appropriate one would be `EditIdentityDepository` ?

Reply:
hehe yes, cleaned up all names